modules/installation-installer-provisioned-vsphere-config-yaml.adoc

```diff
@@ -21,69 +21,103 @@ ifeval::["{context}" == "installing-restricted-networks-vmc"]
 endif::[]
 
 [id="installation-installer-provisioned-vsphere-config-yaml_{context}"]
-= Sample install-config.yaml file for an installer-provisioned VMware vSphere cluster
+= Sample `install-config.yaml` file for an installer-provisioned VMware vSphere cluster
 
-You can customize the install-config.yaml file to specify more details about
-your {product-title} cluster's platform or modify the values of the required
-parameters.
+You can customize the `install-config.yaml` file to specify more details about your {product-title} cluster's platform or modify the values of the required parameters.
+
+[NOTE]
+====
+The {product-title} {product-version} release introduces the `vcenters` and `failureDomains` fields for the `install-config.yaml` file. To define a zonal-enablement configuration for an environment with multiple datacentres running in a single VMware vCenter, you must set values for these fields. 
```

Comment:
Yes, agreed. I applied your recommended sentence. Yeah, the term is a tough one to consider, because no definition term exists for defining a zonal config for vSphere.